cli/commands/site/create.ts

```diff
@@ -289,6 +277,31 @@ export const registerCommand = ( yargs: StudioArgv ) => {
 					type: 'string',
 					describe: __( 'WordPress version (e.g., "latest", "6.4", "6.4.1")' ),
 					default: DEFAULT_VERSIONS.wp,
+					coerce: ( value: string ) => {
```

Comment:
Currently, our validators are spread across `registerCommand->choices`, `runCommand` and `handler`. So instead of using 3 places, I propose to use coerce option as kinda handler for command arguments. This way we will have straightforward structure as:

1. `coerce` is used for validation, type coercion and in general having dedicated place for prepering arguments before starting actual `runCommand`.
2. `runCommand` only for doing command job.

Of course, some validators may be inside `runCommand`, but I propose at least splitting responsibilities as much as possible.